inspector/.env.example

```diff
@@ -0,0 +1,5 @@
+PRIV_KEY=
+BAOBAB_ACC_ID=
+CYPRESS_ACC_ID=
+BAOBAB_GRAPHNODE_DB_CONNECTION_URL=
```

Comment:
Are we using graphnode in this PR? 🤔

Reply:
no but it'll be used in next update, should I remove it and add it from next pr?

Reply:
Yes. Let's first discuss the direction for the inspector. I feel like data feeds liveliness is already tracked by other components, so it might be better to improve those if they are not sufficient.